samples/smoketest/key_vault_secrets_async.py

```diff
@@ -0,0 +1,51 @@
+import os
+from azure.identity.aio import DefaultAzureCredential
+from azure.keyvault.secrets.aio import SecretClient
+
+
+class KeyVault_async:
```

Comment:
I don't think you need to qualify the class names, I don't see a conflict with the sync classes. Also a style point for consideration, you don't need classes here--you could implement these tests with module-level functions.